src/node/rpc/handler_registry.h

```diff
@@ -132,6 +217,7 @@ namespace ccf
       // Default is that all verbs are allowed
       uint64_t allowed_verbs_mask = ~0;
 
+      // https://github.com/microsoft/CCF/issues/1102
       Handler& set_allowed_verbs(std::set<http_method>&& allowed_verbs)
```

Comment:
I haven't documented this not the `ReadWrite` enum as they will most likely be removed in the near future.